### ISSUE TITLE
Quick Edit Failure: Ajax Call returns -1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#97]
```
What steps will reproduce the problem?
1. Create a new Custom Content Post
2. view the content posts in wp-admin
3. click on Quick Edit for a post 
4. Click Update

What is the expected output? What do you see instead?

I expect the spinning ajax wheel to disappear, the row to flash, disappear and 
re-appear. The new row is an updated version that takes into account what was 
changed in Quick Edit.

However, I do not see this. The ajax spinning wheel disappears but a -1 is 
displayed in the row and the row is never reloaded.

What version of the plugin are you using? 0.9.0
What version of WordPress? 3.1.3

```

Original issue reported on code.google.com by `lefor...@gmail.com` on 6 Jun 2011 at 11:35

Attachments:
- [screen.png](https://storage.googleapis.com/google-code-attachments/wordpress-custom-content-type-manager/issue-97/comment-0/screen.png)

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the detailed report! I'll look into this shortly.
```

Original comment by `fireproofsocks` on 7 Jun 2011 at 6:14
- Changed state: **InReview**

**Comment by GoogleCodeExporter** [comment]
```
Confirmed...
```

Original comment by `fireproofsocks` on 1 Jul 2011 at 4:23
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Hmm... this seems to be an ongoing problem, and I'm not sure it has anything to 
do with CCTM: 
http://wordpress.org/support/topic/quick-edit-error-1?replies=1

But the behavior does seem to be limited to only custom content types (it does 
not occur with posts or pages).  It also does not occur on a custom post type 
that has no standardized custom fields.

So this must have something to do with the standardized fields.
```

Original comment by `fireproofsocks` on 1 Jul 2011 at 4:38

**Comment by GoogleCodeExporter** [comment]
```
Mmm... other than the -1, this seems to be harmless: it only occurs if there 
are standardized custom fields, but it doesn't seem to affect any functionality.
```

Original comment by `fireproofsocks` on 1 Jul 2011 at 4:41
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
This has been resolved in version 0.9.4: I'm almost certain it had to do with 
issue 52.
```

Original comment by `fireproofsocks` on 28 Sep 2011 at 6:48
- Changed state: **Fixed**
